build.sbt

```diff
@@ -7,6 +7,17 @@ import sbtrelease.ReleasePlugin.ReleaseKeys.releaseProcess
 import sbtrelease.ReleaseStateTransformations._
 import sbtrelease.Utilities._
 import sbtunidoc.Plugin.UnidocKeys._
+import ScoverageSbtPlugin._
+
+lazy val scoverageSettings = Seq(
+  ScoverageKeys.coverageMinimum := 60,
+  ScoverageKeys.coverageFailOnMinimum := false,
+  ScoverageKeys.coverageHighlighting := {
+    if (scalaBinaryVersion.value == "2.10") false
+    else true
+  },
+  ScoverageSbtPlugin.ScoverageKeys.coverageExcludedPackages := "cats\\.bench\\..*"
```

Comment:
Yes - I saw that right after I did the PR - why can one stare at something and only see things by pressing return? :)